spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -74,4 +74,66 @@
     include_examples 'Unauthenticated User'
   end
 
+  describe 'GET #info' do
+    subject(:http_request) { get :info, params: params}
+    
+
+    context 'When user is authenticated' do
+      include_context 'Authenticated User'
+
+      context 'When fetching an existing book' do
+        before do
+          open_library_stubbed_service = instance_double(OpenLibraryService)
+          allow(open_library_stubbed_service).to receive(:book_info).and_return(
+            JSON.parse(File.read('spec/support/fixtures/open_library_service_parsed_response.json')).symbolize_keys)
```

Comment:
Have you rubocop running on spec folder?

Reply:
No, I just configured travis to run it in spec folder too. Because of that I had to change a lot of spec files 😥

Reply:
Next you will not forgot it ;)